linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -52,15 +65,15 @@ var RouterClient = (function() {
     return summary;
   }
 
-  function timeseriesParams() {
+  function timeseriesParams(name) {
     return {
-      strokeStyle: colorLookup[colorIdx++ % colorLookup.length], //TODO: #242 per client color scheme
+      strokeStyle: chartLegend[name.replace("request_latency_ms.", "")],
```

Comment:
I prefer that we don't try parsing metrics in multiple places. Consider something like:
`strokeStyle: chartLegend[name.match(Query.clientQuery().build())[1]]`